src/vivarium_gates_child_iv_iron/components/observers.py

```diff
@@ -27,42 +28,42 @@ def register_stratifications(self, builder: Builder) -> None:
         """Register each desired stratification with calls to _setup_stratification"""
         super().register_stratifications(builder)
 
-        # self.setup_stratification(
-        #     builder,
-        #     name="wasting_state",
-        #     sources=[Source(f'{data_keys.WASTING.name}.exposure', SourceType.PIPELINE)],
-        #     categories={category.value for category in data_keys.CGFCategories},
-        #     mapper=self.child_growth_risk_factor_stratification_mapper,
-        # )
-        #
-        # self.setup_stratification(
-        #     builder,
-        #     name="stunting_state",
-        #     sources=[Source(f'{data_keys.STUNTING.name}.exposure', SourceType.PIPELINE)],
-        #     categories={category.value for category in data_keys.CGFCategories},
-        #     mapper=self.child_growth_risk_factor_stratification_mapper,
-        # )
-        #
-        # self.setup_stratification(
-        #     builder,
-        #     name="maternal_supplementation",
-        #     sources=[Source("maternal_supplementation_exposure", SourceType.COLUMN)],
-        #     categories={"uncovered", "ifa", "mms", "bep"},
-        # )
-        #
-        # self.setup_stratification(
-        #     builder,
-        #     name="iv_iron",
-        #     sources=[Source("iv_iron_exposure", SourceType.COLUMN)],
-        #     categories={"uncovered", "covered"},
-        # )
-        #
-        # self.setup_stratification(
-        #     builder,
-        #     name="bmi_anemia",
-        #     sources=[Source("maternal_bmi_anemia_exposure", SourceType.COLUMN)],
-        #     categories={"cat4", "cat3", "cat2", "cat1"},
-        # )
+        self.setup_stratification(
```

Comment:
Are these intended to be uncommented now? Were they commented out before just for testing?

Reply:
This was an earlier commit in the branch I branched off of, but the runs have had this uncommented for a while now.